recipes/vs2008_runtime/bld.bat

```diff
@@ -0,0 +1,25 @@
+if "%ARCH%" == "32" (set "ARCH_DIR=x86")
+if "%ARCH%" == "64" (set "ARCH_DIR=amd64")
+
+dir /a:d /o:-n /s /b C:\Windows\WinSxS\%ARCH_DIR%_microsoft.vc90.openmp* > vcomp90_locs.txt
```

Comment:
Extracts all matches for VC9 OpenMP libraries on the correct architecture. Sorts them in reverse order by name as it appears that after some constant hash the version number is included. We would like the latest version at the top of the list so it is easy to retrieve. Also make sure that the full directory path is included and no fluff; so, it is easier to extract paths from the results. Make sure only directories are included so that the other metadata-like content with similar prefix is skipped. Dump this to a file, which is easier to load into a variable.